blog/_posts/2017-08-14-with-love-team-bundledore-india.md

```diff
@@ -0,0 +1,77 @@
+title: With love, Team Bundledore, India !
```

Comment:
Yeah, just follow the syntax for categories as other blog post. Otherwise looks fine I guess.